contrib/conan/recipes/libunwindstack/conanfile.py

```diff
@@ -19,7 +19,7 @@ def config_options(self):
             del self.options.fPIC
 
     def source(self):
-        self.run("git clone https://android.googlesource.com/platform/system/core.git android-core")
```

Comment:
tbh, I do not think we should do that.. It is better to fetch it from the initial location.

Reply:
Can you explain? I thought this is an official mirror, isn't it?

Reply:
Anyway. I changed it back. Came to the same conclusion. This is not really fixing things. We need the CI to build our stuff and then the problem is gone.